### DISCUSSION TITLE
As you know, there are shortcuts in settings menu.

### DISCUSSION THREAD

**Discussion by Githubfanboy722** [File-New-Project/EarTrumpet#1597] [Ideas]
However, we can not change output device (like speaker/headset etc) or input device (like microphone1/mic2 ) etc with just one shortcut like (ctrl + 1 ) etc. If this new feature is released, we users will be appreciated.

**Comment by riverar** [comment]
Hi @Githubfanboy722, try out EarTrumpet Actions in the dev builds (https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller). This will let you create the shortcuts you want!

![image](https://github.com/user-attachments/assets/f651c207-4c2b-4328-86aa-1591543abf01)
